### ISSUE TITLE
How to install this module.

### ISSUE THREAD

**Issue by jackcusac0003** [harrigo/EverCrumbs#3]
Can you tell me some install manual so I can install it. When I install it it causes several errors. May be I was doing wrong.

**Comment by shivsingh7150** [comment]
yes, same i have also need help. please provide composer and manually install methods
